### DISCUSSION TITLE
write_result() missing 1 required positional argument: 'options'

### DISCUSSION THREAD

**Discussion by Kinggibbon1987** [dmarx/video-killed-the-radio-star#130] [Q&A]
When I Infer speech from audio, I keep getting this error. Anyone know how to fix this ? This happens with and without whisper_seg checked

![image](https://user-images.githubusercontent.com/111515719/231812393-86387cd5-b663-4c92-be73-6eb9f95876d0.png)

**Comment by elixlary** [comment]
getting the same error

**Reply by Maldussus** [reply]
Same here

**Comment by mikeaschroth** [comment]
I am also getting the same error.

**Comment by Maldus512** [comment]
It's likely due to [this](https://github.com/openai/whisper/commit/43940fc9780cb91c4f94899755b4648e19d7b977#diff-5c8d8f7b721e9e3afc8c51d4e7e6251f73c33ffd28063a8d4a5de4cace68b02b) commit of the whisper library that added the `options` parameter; since the notebook installs the library from the Github master it's vulnerable to breaking changes.

It can probably be fixed by specifying a precise commit (the one before looks good) in the installation step:
```
!pip install git+https://github.com/openai/whisper@255887f219e6b632bc1a6aac1caf28eecfca1bac
# instead of this
# !pip install git+https://github.com/openai/whisper
```
